app/Registration.php

```diff
@@ -69,6 +70,28 @@ public function centre()
         return $this->belongsTo('App\Centre');
     }
 
+    /**
+     * Get the first un-disbursed bundle on a Registration for any Centre.
+     * There should only be one... else make one if there are none.
+     *
+     * @return Model
+     */
+    public function currentBundle()
+    {
+        return $this->bundles()
+            ->where('disbursed_at', null)
+            ->orderBy('id', 'asc')
+            ->firstOrCreate(
+                [
+//                    "allocating_centre_id" =>  Auth::user()->centre,
```

Comment:
Do you want this in or out?

Reply:
I'm undecided. I think I want it out. keeping it for if tests go horribly wrong